docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -122,6 +122,18 @@ However, repeated crashes should be investigated as they may be caused by an und
 
 *Further Reading*: https://docs.couchbase.com/server/current/learn/services-and-indexes/services/analytics-service.html[Analytics Service]
 
+[#CB90056]
+=== Memcached Connections Rejected (CB90056)
+
+*Background*: If Memcached has too many open connections then it won't allow any new ones to be made.
+This will result in client errors on applications attempting to connect to Couchbase Server
+
+*Condition*: The number of open connections to memcached reaches its limit.
```

Comment:
Wrapping technical terms in \`backticks\` makes the spell checker ignore them.

Reply:
👍